### DISCUSSION TITLE
Config with LDO Orbiter Extruder

### DISCUSSION THREAD

**Discussion by julien-printerbox** [dz0ny/klipper-prusa-mk3s#58] [Q&A]
Hi,
Does anybody know what I have to modify to get the right config for LDO Orbiter extruder ?
Thanks in advance.
Julien

**Comment by atom2ueki** [comment]
you have to check orbiter docs not here, this repo is just convert MK3S/MK3S+ to kipper with RP

**Reply by julien-printerbox** [reply]
Yes, I understand, I have a Prusa modified with an orbiter extruder...
